YOLayout/UIView+YOLayout.m

```diff
@@ -0,0 +1,119 @@
+//
+//  UIView+YOLayout.m
+//  YOLayoutExample
+//
+//  Created by Lucas Yan on 8/1/16.
+//  Copyright © 2016 YOLayout. All rights reserved.
+//
+
+#import "UIView+YOLayout.h"
+#import <objc/runtime.h>
+#import "YOLayout.h"
+#import "YOCGUtils.h"
+
+static char kAssociatedLayoutKey;
```

Comment:
Didn't realize `objc_getAssociatedObject` could take an arbitrary pointer as a key. I've always used a string, but this works. Interesting!

Maybe include UIView in the key?

`static char kUIViewAssociatedLayoutKey`